examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1960,9 +1960,20 @@ class GLTFWriter {
 
 		for ( let i = 0; i < objects.length; i ++ ) {
 
+			const object = objects[ i ];
+
 			// We push directly to children instead of calling `add` to prevent
 			// modify the .parent and break its original scene and hierarchy
-			scene.children.push( objects[ i ] );
+
+			if ( object.isGroup ) {
+
+				scene.children.push( ...object.children );
```

Comment:
Hmm, does this work? I suppose it might. I was assuming you might put the logic down in line 2003 so that a group gets treated as a normal glTF scene (rather than an AuxScene). I guess it's pretty much equivalent.